package.json

```diff
@@ -28,7 +28,7 @@
     "clean:modules": "del ./**/node_modules",
     "clean": "del ./dist",
     "start": "yarn angular:dev:start",
-    "build": "npm-run-all build:ci",
+    "build": "npm-run-all build:ci build:website",
```

Comment:
I'm not sure about adding this here. imo we should separate lib builds from website builds because we have the gitub action that builds and deploys website. This includes seperate tasks for vuepress, sb:angular & sb:core. The separation of website/storybook-ng/storybook-core lets us see when one of them fails.

Reply:
Oops sorry I meant to leave this change out of this PR. The `build` task is not called by the CI. Github calls [`build:ci` instead](https://github.com/vmware/clarity/blob/master/.github/workflows/build.yml#L15). Before the old site was removed from the repo the [`build` script was just a short cut](https://github.com/vmware/clarity/blob/ec782d2859e1a0e499c12acd1376d6027e1ac846/package.json#L10 ) to run the `build:ci` and `build:website` commands. It was helpful for devs starting out in the repo. As long as they ran `build` and it passes then its unlikely the CI would fail. I can pull it out of this commit tho so the git history is clean.

Reply:
Reverted the script change 👍